### PR TITLE
update mentor form, and question box component

### PR DESCRIPTION
**Pull request by apanjain**

### CODE REVIEW

**Comment by apanjain** [comment]
up down arrows have already been removed. Check line 14-24 in styles/App.scss

**Comment by hacksdump** [comment]
> up down arrows have already been removed. Check line 14-24 in styles/App.scss

I was referring to keyboard buttons (not GUI buttons).

**Comment by apanjain** [comment]
> > up down arrows have already been removed. Check line 14-24 in styles/App.scss
> 
> I was referring to keyboard buttons (not GUI buttons).

Done

File: frontend/src/routes/DataCollection/MentorForm.js [hunk 1]
```diff
@@ -64,6 +64,11 @@ class MentorForm extends Component {
   toggleCropper = () => {
     this.setState({ cropperToggle: !this.state.cropperToggle });
   };
+  checkKey = (e) => {
+    if (e.which === 38 || e.which === 40) {
```
Comment (hacksdump) [inline frontend/src/routes/DataCollection/MentorForm.js:68]: Add constants in this function for arrow keys.
`const downButtonCode = 40;`

**Pull request merged**